tests/by-util/test_hashsum.rs

```diff
@@ -338,6 +365,29 @@ fn test_check_md5sum() {
     }
 }
 
+#[test]
+fn test_check_md5sum_not_enough_space() {
```

Comment:
I'm not sure whether this test is correct. Doing the steps manually with GNU `md5sum` doesn't lead to an error.

Reply:
yeah, I am fixing this :)